commit ab08ab20f1a01775e91c8e8bcd9a76a23f1c8e9f (refs/changes/99/4899/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-10-01T17:10:56+00:00 (1 year ago)
    
    TRITON-802 Use of #master in dependencies does not lead to reproducible builds
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/package.json b/package.json
index 16904d4..2f079ba 100644
--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#master",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#45a8d7c367300d3003838a4e0241915cf89fe3f9",
     "deep-equal": "1.0.1",
     "dox": "0.9.0",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
